drivers/hmis/app/models/hmis/form/occurrence_point_form_collection.rb

```diff
@@ -0,0 +1,109 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+###
+# Hmis::Form::OccurrencePointFormCollection
+#
+# This class is responsible for determining which Occurrence Point forms to display on a given Enrollment in HMIS.
+# The Occurrence Point forms appear in the "Enrollment Details" card on the HMIS Enrollment dashboard.
+#
+# These forms collect data elements onto an Enrollment "at occurrence" (a.k.a. when they occur),
+# as opposed to data elements that are collected at a specific point in time (e.g. at intake, exit).
+###
+class Hmis::Form::OccurrencePointFormCollection
+  # Struct that backs Types::HmisSchema::OccurrencePointForm
+  OccurrencePointForm = Struct.new(:definition, :legacy, :data_collected_about, keyword_init: true)
```

Comment:
this looks like a private class, maybe signal that it's only instantiated internally?
```suggestion
  OccurrencePointForm = Struct.new(:definition, :legacy, :data_collected_about, keyword_init: true)
  private_constant :OccurrencePointForm
```

Reply:
nice thanks!